library/src/main/scala/com/chatwork/scala/jwk/ECJWK.scala

```diff
@@ -62,7 +63,8 @@ object ECJWK extends ECJWKJsonImplicits {
           x509CertificateChain,
           d,
           privateKey,
-          expireAt
+          expireAt,
+          keyStore
```

Comment:
This one is an unrelated bug I found through a compiler warning that `keyStore` was unused. Might make sense to turn on fatal warnings so that these show up as errors.